### PR TITLE
Resolved - Generalize the function get_path_value_from_dict in kubernetes and json handler and move it to common.py #145

### PR DESCRIPTION
**Pull request by AkashS0510**
#145

Refactored get_path_value_from_dict.
Moved get_path_value_from_dict to common.py.
Refactored Kubernetes and JSON handler files to import from common.py.

### CODE REVIEW

File: src/tirith/providers/common.py [hunk 1]
```diff
@@ -5,6 +5,11 @@ def create_result_dict(value=None, meta=None, err=None) -> Dict:
     return dict(value=value, meta=meta, err=err)
 
 
+def get_path_value_from_dict(key_path, input_dict,get_path_value_from_dict_func):
```
Comment (refeed) [inline src/tirith/providers/common.py:8]: Missing one space on the third param (sorry for the nitpicky :))
I'd also add parameters types while making this change, e.g. `key_path: str`

**Review by refeed — CHANGES_REQUESTED** [review]
Thanks a lot for this! Looks good overall, just one comment

File: src/tirith/providers/common.py (hunk 1 above)
Comment (Akash-CGF) [reply src/tirith/providers/common.py:8]: Heyy, sure. I'll make these changes.

Comment (AkashS0510) [reply src/tirith/providers/common.py]: Heyy @refeed , Have made the changes :)

**Review by samriddhi99 — APPROVED** [review]
Thanks for the changes. LGTM :+1:

**Pull request merged**